Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN set -eux \
  && apt-get update \
  && apt-get install --no-install-recommends --no-install-suggests -y \
     build-essential postgresql-client libmysqlclient-dev mysql-client python3-pip python3-dev \
-    git libxml2-dev libxslt-dev zlib1g-dev gnupg curl lsb-release libpq-dev dnsutils vim && \
+    git libxml2-dev libxslt-dev zlib1g-dev gnupg curl lsb-release libpq-dev dnsutils vim libmagic-dev && \
```

Comment:
Do we need magic or does django now?

Reply:
I'm not sure why this is missing, but it seems to be required for Label Stduio.